bird1-openwrt/Makefile

```
@@ -27,7 +16,7 @@ PKG_NAME := $(BIRD1_PKG)-openwrt
 PKG_IPV4_NAME := $(BIRD1_IPV4_PKG)-openwrt
 PKG_IPV6_NAME := $(BIRD1_IPV6_PKG)-openwrt
 PKG_VERSION := 0.3
-PKG_RELEASE := 2
+PKG_RELEASE := 3
 PKG_BUILD_DIR := $(BUILD_DIR)/$(PKG_NAME)
 PKG_LICENSE := GPL-3.0+
```

Previously I meant this, but your change, which you did at the beginning of this file, is also good. :+1: 

```suggestion
PKG_LICENSE :=GPL-3.0-or-later
```

We are getting to it! :) I think that we should also remove the indentation on the 2nd row in post-install script, but once CI is merged https://github.com/openwrt/routing/pull/954, it will tell us. :)

---

Ah, cool! I'm changing it right now. :+1: 

True, the rest of packages do not have that indentation, I'm removing it too.

Thanks a lot!